### DISCUSSION TITLE
is kitty faster than the default terminal.app on apple silicon?

### DISCUSSION THREAD

**Discussion by DeveloperKev** [kovidgoyal/kitty#5449] [Q&A]
Hi everyone,
i am a daily user of kitty in linux. switching to mac soon. so i was wondering if i could get answers beforehand.
thanks

**Comment by DeveloperKev** [comment]
@kovidgoyal sorry for the ping.

**Comment by page-down** [comment]
> ... daily user of kitty ... switching to mac soon ...

All I can say is that since you're already using it, the feature-rich nature and ease of use of kitty will outweigh the subtle speed differences.
(I don't use Terminal.app, so I don't know the speed difference, and I don't believe kitty runs slow.)

More than the unnoticeable speed difference, you may need to be aware of the poor environment of macOS and the special configurations to fight it.
